scripts/windows-setup.ps1

```diff
@@ -0,0 +1,159 @@
+Configure "kguiaddons" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-kguiaddons" --config Debug --target install
+
+# Build KWidgetsAddons
+Clone "kwidgetsaddons" "https://invent.kde.org/frameworks/kwidgetsaddons.git"
+Configure "kwidgetsaddons" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-kwidgetsaddons" --config Debug --target install
+
+# Build KColorScheme
+Clone "kcolorscheme" "https://invent.kde.org/frameworks/kcolorscheme.git"
+Configure "kcolorscheme" "-DBUILD_TESTING=OFF"
+# Workaround for Windows
+(Get-Content -ReadCount 0 "$PrefixDir/lib/cmake/KF6I18n/build-pofiles.cmake") -replace 'FATAL_ERROR', 'WARNING' | Set-Content "$PrefixDir/lib/cmake/KF6I18n/build-pofiles.cmake"
+cmake --build "$BuildDir-kcolorscheme" --config Debug --target install
+
+# Build KConfigWidgets
+Clone "kconfigwidgets" "https://invent.kde.org/frameworks/kconfigwidgets.git"
+Configure "kconfigwidgets" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-kconfigwidgets" --config Debug --target install
+
+# Build KIconThemes
+Clone "kiconthemes" "https://invent.kde.org/frameworks/kiconthemes.git"
+Configure "kiconthemes" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-kiconthemes" --config Debug --target install
+
+# Build Sonnet
+Clone "sonnet" "https://invent.kde.org/frameworks/sonnet.git"
+Configure "sonnet" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-sonnet" --config Debug --target install
+
+# Build KCompletion
+Clone "kcompletion" "https://invent.kde.org/frameworks/kcompletion.git"
+Configure "kcompletion" "-DBUILD_TESTING=OFF"
+cmake --build "$BuildDir-kcompletion" --config Debug --target install
+
+# Build KTextWidgets
+Clone "ktextwidgets" "https://invent.kde.org/frameworks/ktextwidgets.git"
+Configure "ktextwidgets" "-DBUILD_TESTING=OFF" "-DWITH_TEXT_TO_SPEECH=OFF"
+cmake --build "$BuildDir-ktextwidgets" --config Debug --target install
+
+# Build KXmlGui
+Clone "kxmlgui" "https://invent.kde.org/frameworks/kxmlgui.git"
+Configure "kxmlgui" "-DBUILD_TESTING=OFF" "-DFORCE_DISABLE_KGLOBALACCEL=ON"
+cmake --build "$BuildDir-kxmlgui" --config Debug --target install
+
+# Build glm
+Clone "glm" "https://github.com/g-truc/glm.git"
+Configure "glm" "-DGLM_BUILD_TESTS=OFF"
+cmake --build "$BuildDir-glm" --config Debug --target install
+
+# Build Corrosion
+Clone "corrosion" "https://github.com/corrosion-rs/corrosion.git"
+Configure "corrosion" "-DCORROSION_BUILD_TESTS=OFF"
+cmake --build "$BuildDir-corrosion" --config Debug --target install
+
+# Build nlohmann
+Clone "json" "https://github.com/nlohmann/json.git"
+Configure "json" "-DJSON_BuildTests=OFF"
+cmake --build "$BuildDir-json" --config Debug --target install
+
+# Build stb
+Clone "stb" "https://github.com/nothings/stb.git"
+mv $LocalDir/stb/* $PrefixDir/include
+
+# Build SPIRV-Cross
+Clone "SPIRV-Cross" "https://github.com/KhronosGroup/SPIRV-Cross.git"
+Configure "SPIRV-Cross"
+cmake --build "$BuildDir-SPIRV-Cross" --config Debug --target install
```

Comment:
Note that the Vulkan SDK actually ships with these libraries (and also GLM, I forgot when I wrote the action.) I'm OK with building them again here because I find finding these libraries in their SDK to be super finicky, but we could look into removing that later to save a little bit of build time.

Reply:
Huh, they do? The Vulkan SDK I had installed didn't have these, but maybe I was just on an outdated version.